ci-operator/jobs/operator-framework/community-operators/operator-framework-community-operators-master-presubmits.yaml

```diff
@@ -0,0 +1,57 @@
+presubmits:
+  operator-framework/community-operators:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/scorecard
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    name: pull-ci-operator-framework-community-operators-master-scorecard
+    rerun_command: /test scorecard
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
```

Comment:
Do we want PR authors to have access to the namespace? Is it possible to only have folks in the OWNERS to have access? We just need the PR authors to know what exactly failed.

Reply:
+1 otherwise only pr authors (who may have no idea how to check the cluster) will be able to access this. it would be good to investigate how to give the OWNERS access to this as well.